bloom-housing/bloom
# refactor: uptake and merge card

# Pull Request Template

## Issue Overview

This PR addresses #3904


## Description

This PR includes a refactor to the Card components that ended up being required to do the passwordless toggle on the sign-in page cleanly. 

It (1) fixes some bugs that were introduced in mandated accounts, (2) abstracts the AccountCard component into a BloomCard component, and (3) uptakes the BloomCard in the application as well so that we can have consistent padding on our form cards throughout the public site. 

It also uptakes BloomCard on the two remaining account screens that still had the UIC FormCard - forgot password and reset password.

## How Can This Be Tested/Reviewed?

Known changes are [here](https://docs.google.com/document/d/1jaqWzAQUkD11th_dodXnPXi8SFGS4-iDoo0-XNZf1Pc/edit?usp=sharing)

And there should be no regressions on:
* The create account page
* The account edit page
* The my applications page
* The account dashboard

I'd recommend testing on the [deploy preview](https://deploy-preview-3922--bloom-exygy-dev.netlify.app/)

## Checklist:

- [ ] I have added QA notes to the issue with applicable URLs (this is just a stepping stone)

## Reviewer Notes:

Steps to review a PR:

- Read and understand the issue, and ensure the author has added QA notes
- Review the code itself from a style point of view
- Pull the changes down locally and test that the acceptance criteria is met
- Also review the acceptance criteria on the Netlify deploy preview (noting that these do not yet include any backend changes made in the PR)
- Either explicitly ask a clarifying question, request changes, or approve the PR if there are small remaining changes but the PR is otherwise good to go

## On Merge:

If you have one commit and message, squash. If you need each message to be applied, rebase and merge.
